src/lib.rs

```diff
@@ -76,14 +74,18 @@
 #![no_std]
 
 mod key_variants;
+mod keys;
 
 #[cfg(feature = "alloc")]
 pub mod gadgets;
 
 #[deprecated(note = "Please use DoubleSignature instead")]
 pub type Proof = key_variants::double_key::Signature;
 
+use dusk_jubjub::{JubJubAffine, JubJubExtended, JubJubScalar};
```

Comment:
I think it's always nicer to only add the imports in the files where they are actually used.